packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -679,7 +678,8 @@ contract VotingV2 is
     ) public override onlyIfNotMigrated() {
         uint256 currentRoundId = voteTiming.computeCurrentRoundId(getCurrentTime());
         _freezeRoundVariables(currentRoundId);
-        _updateTrackers(msg.sender);
+        address voter = delegateToStaker[msg.sender] != address(0) ? delegateToStaker[msg.sender] : msg.sender;
```

Comment:
nit: maybe this could be a `getVoter()` function if we are reusing this logic

Reply:
good call. have refactored